services/src/WebAPI/Controllers/User/UserQueryController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Application.Models;
+using Application.Models.Pagination;
+using Application.Users.Queries.GetAllUsers;
+using Application.Users.Queries.GetLoggedUser;
+using Application.Users.Queries.GetUser;
+using Domain.Common.Pagination.Response;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using WebAPI.Common.Authorization;
+
+namespace WebAPI.Controllers.User;
+
+[ApiController]
+[Route("api/v1/users")]
+
+
+public class UserQueryController : ControllerBase
+{
+    private readonly IMediator _mediator;
+
+    public UserQueryController(IMediator mediator)
+    {
+        _mediator = mediator;
+        
+    }
+    
+    [HttpGet]
+    [AuthorizeAdmin]
+    public async Task<ActionResult<IEnumerable<UserDetailsDto>>> GetInfoAboutUser([FromQuery] string pageRequestDto)
```

Comment:
1. pageRequestDto shouldn't be a string, you could change string to PageRequestDto class
2. this endpoint returns always only one user so there is no reason for using PageRequestDto
3. Why are you returning IEnumerable when there will always be only one user returned? This works but it is misleading and should be fixed
4. you should change `FromQuery` to `FromRoute` and change argument name to `username`